docs/pipelines/core/contextual-matcher.md

```diff
@@ -110,6 +112,36 @@ In this case, the configuration can be concatenated in a list:
 patterns = [cancer, lymphome]
 ```
 
+## Available parameters for more flexibility
+
+3 main parameters can be used to refine how entities will be formed
+
+### The `include_assigned` parameter
+
+Following the previous example, you might want your extracted entities to **include**, if found, the cancer stage and the metastasis status. This can be achieved by setting `include_assigned=True` in the pipe configuration.
+
+For instance, from the sentence "Le patient a un cancer au stade 3", the extracted entity will be:
+
+- "cancer" if `include_assigned=True`
+- "cancer au stade 3" if `include_assigned=True`
+
+### The `reduce_mode` parameter
+
+It might happend that an assign matches multiple times: For instance, in the (non-sensical) sentence "Le patient a un cancer au stade 3 et au stade 4", both "stade 3" and "stade 4" will be matched by the `stage` assign key. Depending on your use case, you may want to keep all extractions, or only a signle one.
```

Comment:
```suggestion
It may happen that an assignment matches more than once. For instance, in the (nonsensical) sentence "Le patient a un cancer au stade 3 et au stade 4", both "stade 3" and "stade 4" will be matched by the `stage` assign key. Depending on your use case, you may want to keep all the extractions, or just one.
```